share/pot/stop.sh

```diff
@@ -100,6 +100,14 @@ _js_stop()
 		pkill -f "$_pdir/ncat-$_pname"
 	fi
 
+	# Garbage collect POSIX shared memory
+	if command -v posixshmcontrol >/dev/null; then
+		_shm_paths=$( posixshmcontrol ls | cut -f 5 | grep "^$_pdir" )
```

Comment:
The regex is ambiguous.
Imagine two `pot`s named `foo` and `foobar`: `_pdir` of `foo` will also match `foobar`
The easy fix is to add the expected suffix `/`

Reply:
Good catch